### PR TITLE
My first PR

### PR DESCRIPTION
**Pull request by ArpitShukIa**
Added a new hello world heading.

### CODE REVIEW

**Comment by icproject01** [comment]
Looks good to me!

**Comment by icproject01** [comment]
We needed a paragraph here.

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <title>Document</title>
 </head>
 <body>
+    <h1>Hello World</h1>
     <div class="body">
```
Comment (icproject01) [inline index.html]: This would look good with paragraph.

Comment (ArpitShukIa) [reply index.html]: Done

**Pull request merged**